project/src/main/java/com/google/sps/data/Restaurant.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.data;
+
+import com.google.auto.value.AutoValue;
+import java.util.List;
+import java.util.Map;
+
+@AutoValue
+public abstract class Restaurant {
+  public abstract String getName();
+  public abstract String getAddress();
+  public abstract Map<String, Double> getLatLngCoords();
+  public abstract double getAvgRating();
+  public abstract int getNumRatings();
+  public abstract int getPriceLevel();
+  public abstract String getId();
```

Comment:
very nit: can we group the id with other string fields (so put it before name perhaps)?